frontend/src/components/common/export-recipients/ExportRecipients.tsx

```diff
@@ -48,19 +55,42 @@ const ExportRecipients = ({
         'text/csv'
       )
     } catch (error) {
-      console.log(error)
+      console.error(error)
     } finally {
       setDisabled(false)
     }
   }
 
+  function renderExportButton() {
+    if (exportStatus === CampaignExportStatus.NoError) {
+      return (
+        <span
+          className={cx(styles.noError, { [styles.alignLabel]: alignLabel })}
```

Comment:
is this alignLabel prop necessary?

Reply:
This is used to align the `No error` status in the campaign list component, since it doesn't have an icon and the other two statuses contain icons, I used a margin to align the text similar to the other statuses

Reply:
Seems misaligned when there are no other statuses.

![Screenshot 2020-07-23 at 10 37 51 PM](https://user-images.githubusercontent.com/10072985/88299682-3981f380-cd35-11ea-99c5-a518ed77e464.png)

Reply:
On the other hand, centre-aligned seems ok for any scenario
![Screenshot 2020-07-23 at 10 42 24 PM](https://user-images.githubusercontent.com/10072985/88300208-cd53bf80-cd35-11ea-9955-a1bd8ba94fd3.png)

Reply:
@sarahsalim thoughts?

Reply:
@pregnantboy @miazima Sure, centre-aligned is alright.